Shared/Samples/Find route in mobile map package/FindRouteInMobileMapPackageView.swift

```diff
@@ -101,23 +101,19 @@ private extension FindRouteInMobileMapPackageView {
                         .resizable()
                         .scaledToFit()
                         .frame(height: 50)
-                        .overlay {
+                        .overlay(alignment: .top) {
                             // The symbols indicating the map's functionality.
-                            VStack {
-                                HStack {
-                                    if !map.transportationNetworks.isEmpty {
-                                        // The symbol indicating whether the map can route.
-                                        Image(systemName: "arrow.triangle.turn.up.right.circle")
-                                    }
-                                    Spacer()
-                                    if mapPackage.locatorTask != nil {
-                                        // The symbol indicating whether the map can geocode.
-                                        Image(systemName: "mappin.circle")
-                                    }
-                                }
-                                .padding(2)
+                            HStack {
+                                // The symbol indicating whether the map can route.
+                                Image(systemName: "arrow.triangle.turn.up.right.circle")
+                                    .opacity(!map.transportationNetworks.isEmpty ? 1 : 0)
                                 Spacer()
+                                // The symbol indicating whether the map can geocode.
+                                Image(systemName: "mappin.circle")
+                                    .opacity(mapPackage.locatorTask != nil ? 1 : 0)
                             }
+                            .foregroundColor(.black)
+                            .padding(2)
```

Comment:
while this works, I have 2 issues with it:
- doesn't look aesthetically great to my eye
- hard-coding a color (black) means that it works with the map in the sample, but if we change that to a darker map it will not work 

I prefer this:

```swift
label: {
                    HStack {
                        // The image of the map for the row.
                        Image(uiImage: map.item?.thumbnail?.image ?? UIImage(
                            systemName: "questionmark"
                        )!)
                        .resizable()
                        .scaledToFit()
                        .frame(height: 50)
                        VStack(alignment: .leading) {
                            Text(mapName)
                            // The symbols indicating the map's functionality.
                            HStack {
                                if !map.transportationNetworks.isEmpty {
                                    // The symbol indicating whether the map can route.
                                    Image(systemName: "arrow.triangle.turn.up.right.circle")
                                        .imageScale(.small)
                                }
                                // The symbol indicating whether the map can geocode.
                                if mapPackage.locatorTask != nil {
                                    Image(systemName: "mappin.circle")
                                        .imageScale(.small)
                                }
                            }
                            .foregroundStyle(.secondary)
                        }
                    }
                }
```

which looks like this:

<img width="698" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/3718875/5d5c7e3a-2642-4616-a721-8200c90be4ee">

<img width="698" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/3718875/0215e313-7975-40a0-9f7d-6fd0ad4a5657">

Reply:
This is an interesting idea that we didn't think of. It resembles the "nutrition labels" for app on App Store. We may actually expand the icons into secondary Labels in an HStack that looks like
```
-----------------------
|Transportation       |
|↪️ Supports Routing   |
|📍Supports Geocoding |
-----------------------
```

Reply:
If you want to do that, you can write this:

```swift
                } label: {
                    HStack {
                        // The image of the map for the row.
                        Image(uiImage: map.item?.thumbnail?.image ?? UIImage(
                            systemName: "questionmark"
                        )!)
                        .resizable()
                        .scaledToFit()
                        .frame(height: 50)
                        VStack(alignment: .leading) {
                            Text(mapName)
                            // The symbols indicating the map's functionality.
                            VStack(alignment: .leading) {
                                if !map.transportationNetworks.isEmpty {
                                    HStack(spacing: 4) {
                                        // The symbol indicating whether the map can route.
                                        Image(systemName: "arrow.triangle.turn.up.right.circle")
                                        Text("Supports Routing")
                                    }
                                }
                                // The symbol indicating whether the map can geocode.
                                if mapPackage.locatorTask != nil {
                                    HStack(spacing: 4) {
                                        Image(systemName: "mappin.circle")
                                        Text("Supports Geocoding")
                                    }
                                }
                            }
                            .font(.caption2)
                            .foregroundStyle(.secondary)
                        }
                    }
                }
```
<img width="698" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/3718875/90fc2b8a-5205-4fd1-bd23-3f79da0e48a3">

Reply:
Or something like this:

```swift
                } label: {
                    HStack {
                        // The image of the map for the row.
                        Image(uiImage: map.item?.thumbnail?.image ?? UIImage(
                            systemName: "questionmark"
                        )!)
                        .resizable()
                        .scaledToFit()
                        .frame(height: 50)
                        VStack(alignment: .leading) {
                            Text(mapName)
                            // The symbols indicating the map's functionality.
                            HStack {
                                if !map.transportationNetworks.isEmpty {
                                    HStack(spacing: 2) {
                                        // The symbol indicating whether the map can route.
                                        Image(systemName: "arrow.triangle.turn.up.right.circle")
                                        Text("Routing")
                                    }
                                }
                                // The symbol indicating whether the map can geocode.
                                if mapPackage.locatorTask != nil {
                                    HStack(spacing: 2) {
                                        Image(systemName: "mappin.circle")
                                        Text("Geocoding")
                                    }
                                }
                            }
                            .font(.caption2)
                            .foregroundStyle(.secondary)
                        }
                    }
                }
```

which looks like this:

<img width="698" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/3718875/82c0638c-85e5-4ff2-b7c4-69ab04d47a37">